src/server/services/github/index.js

```diff
@@ -50,6 +50,6 @@ module.exports = {
 }
 
 function logRateLimit(response) {
-  print.trace('Rate limit:', response.data.rateLimit)
+  debug('DD - Rate limit:', response.data.rateLimit)
```

Comment:
@pnkapadia6 what does the `DD -` stands for?

Reply:
Sending another PR with those changes

Reply:
@pnkapadia6 that's fixed, don't worry. You did the hard work already.